legate/core/operation.py

```diff
@@ -151,27 +170,35 @@ def add_alignment(self, store1, store2):
         part2 = self._get_unique_partition(store2)
         self.add_constraint(part1 == part2)
 
-    def add_broadcast(self, store, axes=None):
+    def add_broadcast(
+        self, store: Store, axes: Optional[Union[int, Iterable[int]]] = None
+    ) -> None:
         self._check_store(store)
-        part = self._get_unique_partition(store)
+        part = self._get_unique_partition(store)  # XXXXX
```

Comment:
Looks like you have a similar comment. what is it for?

Reply:
https://github.com/nv-legate/legate.core/pull/189/commits/7bc51b3a159364102d2a8a3fbd2049bbbe585d5c